File: codeforces/go/236A.go
package main

import "fmt"

func main() {
	var s string
	var count int
	l := make([]bool, 26)
	fmt.Scan(&s)
	for _, c := range s {
		l[c - 'a'] = true
	}
	for _, k := range l {
		if k {
			count++
		}
	}
	if count%2 == 0  {
		fmt.Println("CHAT WITH HER!")
	} else {
		fmt.Println("IGNORE HIM!")
	}
}

File: codeforces/c/236A.c
#include<stdio.h>
#include<stdbool.h>
#include<string.h>

int main(void) {
	int count = 0, i;
	char string[100];
	bool letter[26];
	for (i=0; i<26; i++)
		letter[i] = false;
	scanf("%s\n", string);
	for (i=0; i<strlen(string); i++)
		letter[(int)string[i]-97] = true;
	for (i=0; i<26; i++)
		if (letter[i]) count++;
	printf("%s\n", count%2 == 0 ? "CHAT WITH HER!" : "IGNORE HIM!");
	return 0;
}
